src/components/browser/browser.js

```diff
@@ -8,7 +8,7 @@ class Browser extends React.PureComponent {
     return (
       <div className="shadow-darken25 round bg-white">
         <div className="bg-gray h30-mm h18 round-t">
-          <div className="color-white flex-parent flex-parent--center-cross h30-mm h18 px6">
+          <div className="color-white flex flex--center-cross h30-mm h18 px6">
             <span className="color-red-light">
               <Icon name="circle" />
             </span>
```

Comment:
We should consider removing `<Icon name="circle" />` and replacing with HTML/CSS circle. This will reduce the generated markup significantly while delivering the same result. Example:

```suggestion
            <div class="bg-red-light w12 h12 round-full"></div>
```

This change is slightly out of scope, so I will create an issue.

Reply:
Opened issue: https://github.com/mapbox/dr-ui/issues/485